embark.el

```diff
@@ -1148,6 +1196,10 @@ after this delay shows the verbose indicator."
   :type '(choice (const :tag "No delay" 0)
                  (number :tag "Delay in seconds")))
 
+(defcustom embark-mixed-indicator-both t
```

Comment:
Oh, you prefer the repetition Prot complained about?

Reply:
Yes, indeed I do. I also mentioned it to @protesilaos in the discussion of #310, see https://github.com/oantolin/embark/issues/310#issuecomment-889732884.

Reply:
But making it the default setting is of course debatable. I usually think it is easier for users to turn something off than on, since how would they know. It seems to me that these configurations work well as default:

* reuse-window and indicator-both=t
* resizable at bottom and indicator-both=nil

Reply:
OK. Would you consider making the default nil? I only thought removing the minimal indicator looked "jumpy" when the minimal indicator came back while cycling. Now I think the single jump when switching from minimal to verbose is smooth.

Reply:
Even with reuse-window I prefer nil: the sudden appearance of the verbose indicator naturally takes your eyes away from the echo area, so you don't even notice the minimal indicator disappearing. Conversely, if you do keep your eyes on the echo area, the disappearance reminds you the verbose indicator has now appeared elsewhere.

Reply:
Well, this isn't important. We can discuss it once merged.

Reply:
> Would you consider making the default nil?

Ok, done.

> Even with reuse-window I prefer nil: the sudden appearance of the verbose indicator naturally takes your eyes away from the echo area, so you don't even notice the minimal indicator disappearing. Conversely, if you do keep your eyes on the echo area, the disappearance reminds you the verbose indicator has now appeared elsewhere.

For me it is like this - if I keep my eyes focused on the minibuffer or on the element I act on, I notice the vindicator appearing but I don't really look at it, probably in particular when repeating or cycling. The vindicator is still helpful in order to have the help readily available. But the disappearance of the Act symbol is disturbing, e.g., in the minibuffer without quit it looks as if I already acted.